actpool/actioniterator/actioniterator_test.go

```diff
@@ -19,37 +19,16 @@ import (
 func TestActionIterator(t *testing.T) {
```

Comment:
Better to replace the vote action with transfer action here to keep the purpose of this test